### PR TITLE
Media Kit

### PR DESCRIPTION
**Pull request by davidfischer**
- Graphics
- Colors
- Guidelines
- Contact form

I replaced the email in the footer although I wasn't sure about that.
Maybe we should keep it.

### CODE REVIEW

File: content/pages/press.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Title: Media Kit
+slug: press
+description: Are you part of the tech press and you want to write about advertising that doesn't track you? This is the right place.
```
Comment (ericholscher) [inline content/pages/press.md:3]:
```suggestion
description: Are you a journalist that wants to write about advertising that doesn't track you? This is the right place.
```

File: ethicalads-theme/templates/includes/footer.html [hunk 2]
```diff
@@ -149,7 +149,7 @@ <h6 class="font-weight-bold text-uppercase text-gray-700">Company</h6>
             <a href="/privacy-policy/" class="text-reset">Privacy Policy</a>
           </li>
           <li class="mb-3">
-            <a href="mailto:ads@ethicalads.io" class="text-reset">Email us</a>
+            <a href="/press/" class="text-reset">Media Kit</a>
```
Comment (ericholscher) [inline ethicalads-theme/templates/includes/footer.html:152]: These are not the same. The primary use of this "email us" is for people who have questions about becoming advertisers/publishers. We should just have a Contact Us page probably, and include a form?

File: content/pages/press.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+template: ea/press
+
+
+Want to speak with someone at EthicalAds? Please get in touch [below](#inbound-form).
```
Comment (ericholscher) [inline content/pages/press.md:7]: We should include a basic style guide here around how to refer to us. Eg.


```suggestion
Thanks for your interest in writing about EthicalAds & Read the Docs.

### Quick Style Guide

* Our brand name is `EthicalAds` when referencing us. You can use `ethical advertising` to talk about the category or concept. 
* Our parent company is `Read the Docs`, which can be abbreviated at `RTD`.

### Contact us

Want to speak with someone at EthicalAds? Please get in touch [below](#inbound-form).
```

**Review by ericholscher — APPROVED** [review]
Looks like a good start! We should probably beef up the RTD one at some point, but this is good.

File: ethicalads-theme/templates/includes/footer.html (hunk 2 above)
Comment (davidfischer) [reply ethicalads-theme/templates/includes/footer.html:152]: Sounds good. I'll add the email back for now and just build a tiny contact us page.

Comment (davidfischer) [reply ethicalads-theme/templates/includes/footer.html:152]: https://github.com/readthedocs/ethicalads.io/pull/113

**Pull request merged**